<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Mount</class>
 <widget class="QDialog" name="X2Mount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>634</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>634</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setup ATCS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>9999</width>
       <height>9999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>56</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>257</x>
        <y>568</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>568</y>
        <width>81</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>400</width>
        <height>128</height>
       </rect>
      </property>
      <property name="title">
       <string>Settings</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>24</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Mount type :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="enableCordWrap">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>56</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Cord Wrap</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="cordWrapPosition">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>88</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>88</y>
         <width>136</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Cord Wrap Position :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>104</x>
         <y>24</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Alt/Az</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>192</x>
         <y>24</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Equatorial</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>8</y>
        <width>134</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Â© RTI-Zone.org, 2018</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
